File: dsa-crio/DSA-JS-101/Problems/Classes/6. DSA Recusrsion Problem/2. Nth Fibonacci Number/code.ts
const nthFibonnacci = (n: number): number => {
  if (n == 0 || n == 1) return n;

  return nthFibonnacci(n - 1) + nthFibonnacci(n - 2);
};

console.log(nthFibonnacci(8));

File: dsa-striver/02-patterns/13/code.cpp


#include <iostream>
using namespace std;

void stars(int n)
{
    int number = 1;
    for (int i = 0; i <= n; i++)
    {
        for (int j = 0; j < i; j++)
        {
            cout << number++ <<" ";
        }

        cout << endl;
    }
}

int main()
{
    stars(5);
    return 0;
}
